### DISCUSSION TITLE
Windows vs. Mac/Unix

### DISCUSSION THREAD

**Discussion by Sandman-1** [GreenleafLab/ArchR#1140] [Questions / Documentation]
Hello! I am using a windows OS to run ArchR. However, it says on the ArchR website that Windows is not supported. My assumption is that Windows will run fine because I am going through the tutorial with it, but due to the limitation on the number of threads, it can't be used for parallelization. Does this mean it will run slower to complete multiple tasks, or does parallelization mean a different kind of issue?

Best,
Skanda

**Comment by rcorces** [comment]
ArchR is not supported on Windows which means that we (the developers) cannot reply to questions or errors from windows users because the software is not designed to work on those systems.
